pandas/tests/dtypes/test_common.py

```diff
@@ -392,6 +392,48 @@ def test_is_not_unsigned_integer_dtype(dtype):
     assert not com.is_unsigned_integer_dtype(dtype)
 
 
+@pytest.mark.parametrize(
+    "dtype1,dtype2",
+    [
+        ("int_", "int_"),
+        ("int8", "int8"),
+        ("Int8", "Int8"),
+        ("int16", "int16"),
+        ("Int16", "Int16"),
+        ("int32", "int32"),
+        ("Int32", "Int32"),
+        ("int64", "int64"),
+        ("Int64", "Int64"),
+        ("uint8", "uint8"),
+        ("uint16", "uint16"),
+        ("uint32", "uint32"),
+        ("uint64", "uint64"),
+        ("float16", "float16"),
+        ("float32", "float32"),
+        ("Float32", "Float32"),
+        ("float64", "float64"),
+        ("Float64", "Float64"),
+    ],
+)
+def test_all_type_comparison_with_true_result(dtype1, dtype2):
```

Comment:
Your dtypes are always equal, so no need to use 2. Can simply use dtype1 for everything (also rename to dtype in this case)

Reply:
But there are some differences in case, is it not important?

Reply:
the test below is fine, but here everything is equal?